DelvUI/Interface/GeneralElements/CastbarHud.cs

```diff
@@ -55,24 +51,35 @@ public override void DrawChildren(Vector2 origin)
                 return;
             }
 
-            UpdateCurrentCast(out var currentCastTime, out var totalCastTime);
+            UpdateCurrentCast(out float currentCastTime, out float totalCastTime);
             if (totalCastTime == 0)
             {
                 return;
             }
 
-            bool validIcon = LastUsedCast != null && LastUsedCast.IconTexture != null;
+            bool validIcon = LastUsedCast is { IconTexture: { } };
```

Comment:
I hate this with passion. Prob because I'm new to the language, but I would have literally no idea what this meant if I didn't have the original line there.

Not sure if there's a name for this kind of syntax and not sure if you're doing it manually or it's some automatic style settings you have, but I REALLY don't want it lol

Reply:
I don't mind changing it back, I just tend to randomly change stuff like that when thinking about something (bad habit prolly haha). It's just (property) pattern matching anyway, but yeah, != null or is not null has way better readability so ill change it back and add changelog in a bit